docker-compose/compose.yaml

```diff
@@ -222,9 +275,17 @@ services:
       AUDIT_ENTRYSTORAGE_OPENSEARCH_CONNECTOR_USERNAME: admin
       AUDIT_ENTRYSTORAGE_OPENSEARCH_CONNECTOR_PASSWORD: admin
       AUDIT_EVENTINGESTION_URI: activemq:topic:alfresco.repo.event2
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8081/actuator/health/readiness"]
+      interval: 60s
+      timeout: 10s
+      retries: 3
+      start_period: 15s
     depends_on:
-      - activemq
-      - elasticsearch
+      activemq:
+        condition: service_healthy
+      elasticsearch:
+        condition: service_healthy
   digital-workspace:
```

Comment:
strictly speaking both webapps are dependant on repo - but maybe we should avoid to declare explicit dependencies that when not met will cause unwanted restarts - so ok to not have repo here